include/retdec/fileformat/file_format/elf/elf_format.h

```diff
@@ -13,6 +13,7 @@
 
 #include "retdec/fileformat/file_format/file_format.h"
 #include "retdec/fileformat/types/note_section/elf_notes.h"
+#include "retdec/fileformat/types/import_table/elf_import_table.h"
```

Comment:
Does this have to be included here? If not, it is best to hide includes in cpp files.